Update Spanish translations

i3visio opened the pull request:
Update non translated strings and remove commonly unneeded expressions in Spanish translations such as `Por favor,` and similar ones.

erciccione commented:
Hi! Thank you for this update. I know @lh1008 was working at the same file in these days, maybe he can leave a review

ordtrogen commented:
When I did my first PR, I had forgotten to remove type="unfinished" from the translated segments and it looks like you may have done the same :-)

ordtrogen commented:
Oh, and I also forgot to squash my fixes after updating the PR. Just sayin ...

erciccione commented:
@ordtrogen is right. @i3visio if a line is translated remove `type="unfinished"` please :)

lh1008 commented:
Hi,..ok I will take a look at the translation... :)

i3visio commented:
@erciccione Yep! We were not sure if the `type="unfinished"` was needed because there were some translated that still had it. We are updating this translation now and updating it!

lh1008 commented on translations/monero-core_es.ts:
```diff
@@ -310,123 +310,123 @@
         <location filename="../pages/Keys.qml" line="206"/>
         <source>(View Only Wallet -  No mnemonic seed available)</source>
-        <translation type="unfinished"></translation>
+        <translation>(Monedero de solo lectura - No existen semillas mnemotécnicas asociadas)</translation>
```
(Monedero de solo lectura - No existe semilla mnemotécnica disponible)
Singular no plural. La palabra asociada hace referencia a grupo.

lh1008 commented on translations/monero-core_es.ts:
```diff
@@ -459,7 +459,7 @@
         <location filename="../LeftPanel.qml" line="439"/>
         <source>Prove/check</source>
-        <translation type="unfinished"></translation>
+        <translation>Probar o verificar</translation>
```
Probar/verificar

Puntuaciones deben ser iguales, Hay que seguir [las guias :)](https://taiga.getmonero.org/project/erciccione-monero-localization/wiki/translating-the-gui)

i3visio replied:
Oops. La inercia.

lh1008 commented on translations/monero-core_es.ts:
```diff
@@ -524,12 +524,12 @@
         <location filename="../LeftPanel.qml" line="463"/>
         <source>Sign/verify</source>
-        <translation>Firmar/verificar</translation>
+        <translation>Firmar o verificar</translation>
```
Firmar/verificar

Puntuaciones deben ser iguales, Hay que seguir [las guias :)](https://taiga.getmonero.org/project/erciccione-monero-localization/wiki/translating-the-gui)

lh1008 commented on translations/monero-core_es.ts:
```diff
@@ -570,17 +570,17 @@
     <message>
         <location filename="../pages/Mining.qml" line="83"/>
         <source>Mining with your computer helps strengthen the Monero network. The more that people mine, the harder it is for the network to be attacked, and every little bit helps.&lt;br&gt; &lt;br&gt;Mining also gives you a small chance to earn some Monero. Your computer will create hashes looking for block solutions. If you find a block, you will get the associated reward. Good luck!</source>
-        <translation>Minar con tu compudatora ayuda a fortalecer la red Monero. Cuanta más gente minando, más difícil es atacar la red, y cada pequeño esfuerzo cuenta.&lt;br&gt; &lt;br&gt;Minar también le da la pequeña oportunidad de ganar algún Monero. Su computadora creará hashes buscando soluciones a bloques. Si encuentra un bloque, obtendrá la recompensa asociada. Buena suerte!</translation>
```
Le doy un punto a esta traducción y solo le agregaria algunas palabras:
<translation>Minar con tu compudatora ayuda a fortalecer la red Monero. Cuanta más gente minando, más difícil **será** atacar la red, y cada pequeño esfuerzo cuenta.&lt;br&gt; &lt;br&gt;Minar también le da **una** pequeña oportunidad de ganar algún Monero. Su computadora creará hashes buscando soluciones a bloques. Si encuentra un bloque, obtendrá la recompensa asociada. !Buena suerte!</translation>

ordtrogen commented:
Esto es una fuente de terminología que me sirve a menudo. Ne he revisado
vuestra discusión, solo para compartir el enlace. Buen trabajo!

https://www.microsoft.com/en-us/language/Search?&searchTerm=unconfirmed&langID=Spanish&Source=true&productid=All
Products





https://github.com/ordtrogen

2018-02-07 5:00 GMT+01:00 LUIS HERRERA <notifications@github.com>:

> *@lh1008* commented on this pull request.
>
> Dejo estas revisiones para evaluar(aun no término de revisar). Espero
> contribuir en algo. Si tienen dudas por favor compartir lo que piensan. Me
> gustaría escucharlos. :)
> ------------------------------
>
> In translations/monero-core_es.ts
> <https://github.com/monero-project/monero-gui/pull/1109#discussion_r166374145>
> :
>
> >      </message>
>  </context>
>  <context>
>      <name>DaemonManagerDialog</name>
>      <message>
>          <location filename="../components/DaemonManagerDialog.qml" line="93"/>
>          <source>Starting local node in %1 seconds</source>
> -        <translation type="unfinished"></translation>
> +        <translation>Arrancando nodo local en %1 segundos</translation>
>
> Sugiero el cambio de "Arrancando" por "Iniciando".
>
>    - Arrancando sirve como empezar a correr una máquina, en este caso es
>    el nodo, y, sí es una máquina - *Arrancando nodo local en %1 segundos*-
>    arranca la máquina.
>    - Iniciando expresa el punto de partida de algo, constituye la primera
>    parte de una cosa, una acción que se va a ejecutar, algo que se va a
>    manifestar, algo va a ocurrir! jeje - *Iniciando nodo local en %1
>    segundos* -, ya sabemos que es una máquina, pero que va a hacer?
>    Iniciar el nodo. Espero comentarios. :)
>
> ------------------------------
>
> In translations/monero-core_es.ts
> <https://github.com/monero-project/monero-gui/pull/1109#discussion_r166395201>
> :
>
> > @@ -242,12 +242,12 @@
>      <message>
>          <location filename="../components/HistoryTable.qml" line="50"/>
>          <source>Tx key:</source>
> -        <translation>Llave Tx:</translation>
> +        <translation>Llave de Tx:</translation>
>
> Sugiero mantener "*Llave Tx*". Tx hace referencia a Transaction(Tx) =
> Transacción(Tx). Cuando decimos Llave Tx se sobreentiende que estamos
> hablando de la *Llave de la Transacción(Tx)* ó Llave Tx. Decir por
> ejemplo, la Llave Tx es de color azul. Sabemos el color del objeto, como un
> todo, siendo la llave y la Tx azul, unidas. Si le ponemos Llave de Tx,
> decir, la Llave *de* Tx es de color azul, ahí, hay una marca en el
> objeto(Llave) de color azul y descartamos al Tx unido a la Llave (sin Llave
> no hay Tx, y sin Tx no hay Llave). En el whitepaper
> <https://cryptonote.org/whitepaper.pdf> del protocolo de
> monero(cryptonote) se dan las características de una transacción, pág. 6-7,
> 4.3 Unlinkable Payments punto 3. Donde explica como se ejecutan las
> transacciones en la cadena de bloques. Ahí encontré estas palabras "*Tx
> public key*". En este caso, si habría que decir - "Llave pública *de* Tx"
> - porque hay una característica adicional de la Llave Tx, que es pública y
> viceversa cuando es "*Tx private key*"- Llave privada *de* Tx. Quedo
> atento. :)
> ------------------------------
>
> In translations/monero-core_es.ts
> <https://github.com/monero-project/monero-gui/pull/1109#discussion_r166504061>
> :
>
> >      </message>
>      <message>
>          <location filename="../components/HistoryTable.qml" line="51"/>
>          <source>Tx note:</source>
> -        <translation>Nota Tx:</translation>
> +        <translation>Nota de Tx:</translation>
>
> Lo mismo con "Llave Tx". Sugiero dejar como "Nota Tx".
> ------------------------------
>
> In translations/monero-core_es.ts
> <https://github.com/monero-project/monero-gui/pull/1109#discussion_r166504682>
> :
>
> >      </message>
>      <message>
>          <location filename="../components/HistoryTable.qml" line="51"/>
>          <source>Tx note:</source>
> -        <translation>Nota Tx:</translation>
> +        <translation>Nota de Tx:</translation>
>
> En los casos de "Tx ID", si sirve la traducción "ID de Tx", porque el ID
> es un identificador, describe a la transacción, algo que la "Llave" y/o la
> "Nota", no hace.
> ------------------------------
>
> In translations/monero-core_es.ts
> <https://github.com/monero-project/monero-gui/pull/1109#discussion_r166505999>
> :
>
> >      </message>
>      <message>
>          <location filename="../components/HistoryTableMobile.qml" line="152"/>
>          <source>UNCONFIRMED</source>
> -        <translation type="unfinished">NO CONFIRMADO</translation>
> +        <translation>NO CONFIRMADO</translation>
>
> Me gustaria determinar si "UNCONFIRMED" hace referencia a "SIN CONFIRMAR"
> ó a "NO CONFIRMADO".
>
>    - SIN CONFIRMAR hace referencía a una carencia de información, algún
>    faltante.
>    - NO CONFIRMADO hace referencía a una negación.
>
> Qué tipo de acción interna ejecuta el GUI? Niega o carece?
> ------------------------------
>
> In translations/monero-core_es.ts
> <https://github.com/monero-project/monero-gui/pull/1109#discussion_r166506446>
> :
>
> > @@ -282,7 +282,7 @@
>      <message>
>          <location filename="../components/HistoryTable.qml" line="264"/>
>          <source>FAILED</source>
> -        <translation type="unfinished"></translation>
> +        <translation>FALLADO</translation>
>
> Que tal la palabra "FALLÓ" ó "FALLIDO"?
> El sistema falló. The system failed.
> Confirmaciones fallidas. Confirmations failed.
> ------------------------------
>
> In translations/monero-core_es.ts
> <https://github.com/monero-project/monero-gui/pull/1109#discussion_r166508460>
> :
>
> >      </message>
>      <message>
>          <location filename="../pages/Keys.qml" line="206"/>
>          <source>(View Only Wallet -  No mnemonic seed available)</source>
> -        <translation type="unfinished"></translation>
> +        <translation>(Monedero de solo lectura - No existen semillas mnemotécnicas asociadas)</translation>
>
> (Monedero de solo lectura - No existe semilla mnemotécnica disponible)
> Singular no plural. La palabra asociada hace referencia a grupo.
> ------------------------------
>
> In translations/monero-core_es.ts
> <https://github.com/monero-project/monero-gui/pull/1109#discussion_r166509179>
> :
>
> > @@ -459,7 +459,7 @@
>      <message>
>          <location filename="../LeftPanel.qml" line="439"/>
>          <source>Prove/check</source>
> -        <translation type="unfinished"></translation>
> +        <translation>Probar o verificar</translation>
>
> Probar/verificar
>
> Puntuaciones deben ser iguales, Hay que seguir las guias :)
> <https://taiga.getmonero.org/project/erciccione-monero-localization/wiki/translating-the-gui>
> ------------------------------
>
> In translations/monero-core_es.ts
> <https://github.com/monero-project/monero-gui/pull/1109#discussion_r166509588>
> :
>
> >      </message>
>      <message>
>          <location filename="../LeftPanel.qml" line="463"/>
>          <source>Sign/verify</source>
> -        <translation>Firmar/verificar</translation>
> +        <translation>Firmar o verificar</translation>
>
> Firmar/verificar
>
> Puntuaciones deben ser iguales, Hay que seguir las guias :)
> <https://taiga.getmonero.org/project/erciccione-monero-localization/wiki/translating-the-gui>
> ------------------------------
>
> In translations/monero-core_es.ts
> <https://github.com/monero-project/monero-gui/pull/1109#discussion_r166513259>
> :
>
> >      </message>
>      <message>
>          <location filename="../pages/Mining.qml" line="83"/>
>          <source>Mining with your computer helps strengthen the Monero network. The more that people mine, the harder it is for the network to be attacked, and every little bit helps.&lt;br&gt; &lt;br&gt;Mining also gives you a small chance to earn some Monero. Your computer will create hashes looking for block solutions. If you find a block, you will get the associated reward. Good luck!</source>
> -        <translation>Minar con tu compudatora ayuda a fortalecer la red Monero. Cuanta más gente minando, más difícil es atacar la red, y cada pequeño esfuerzo cuenta.&lt;br&gt; &lt;br&gt;Minar también le da la pequeña oportunidad de ganar algún Monero. Su computadora creará hashes buscando soluciones a bloques. Si encuentra un bloque, obtendrá la recompensa asociada. Buena suerte!</translation>
>
> Le doy un punto a esta traducción y solo le agregaria algunas palabras:
> Minar con tu compudatora ayuda a fortalecer la red Monero. Cuanta más
> gente minando, más difícil *será* atacar la red, y cada pequeño esfuerzo
> cuenta.<br> <br>Minar también le da *una* pequeña oportunidad de ganar
> algún Monero. Su computadora creará hashes buscando soluciones a bloques.
> Si encuentra un bloque, obtendrá la recompensa asociada. !Buena suerte!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/monero-project/monero-gui/pull/1109#pullrequestreview-94426097>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AOez62Z0DB5mdVL8BjHdP_G8QBEuHZWXks5tSR_CgaJpZM4R4UsZ>
> .
>

lh1008 commented:
Gracias @ordtrogen, lo tendremos presente, sirve bastante para las traducciones técnicas. :)

lh1008 commented on translations/monero-core_es.ts:
```diff
@@ -600,7 +600,7 @@
         <location filename="../pages/Mining.qml" line="129"/>
         <source>Enable mining when running on battery</source>
-        <translation>Habilitar minado en uso con batería</translation>
+        <translation>Habilitar minado mientras se ejecute con batera batería</translation>
```
Tienes la pabara **batera** hay que quitarla.
Sugiero:
Habilitar minado mientras la bateria *está en uso* o *se esta usando* 

En las traducciones de windows "running" se traduce como "usando". Se que esto no es windows, pero windows ya tiene todo un mercado de lenguaje técnico que podemos aprovechar. @ordtrogen no dio la herramienta para usar :) [Windows search terminology](https://www.microsoft.com/en-us/language/Search?&searchTerm=unconfirmed&langID=Spanish&Source=true&productid=All)

lh1008 commented on translations/monero-core_es.ts:
```diff
@@ -610,27 +610,27 @@
         <location filename="../pages/Mining.qml" line="148"/>
         <source>Start mining</source>
-        <translation>Empezar minado</translation>
+        <translation>Empezar a minar</translation>
```
Empezar minería

Otra opción

lh1008 commented on translations/monero-core_es.ts:
```diff
@@ -610,27 +610,27 @@
         <location filename="../pages/Mining.qml" line="158"/>
         <source>Error starting mining</source>
-        <translation>Error arrancando el minado</translation>
+        <translation>Error arrancando el proceso de minado</translation>
```
Error arrancando la minería
Error arrancando el proceso de minería

Minado es una palabra conjugada en tiempo pasado. Se hace énfasis en el error de minería presente.
él mina - él minó - él ha minado
ella mina - ella minó - ella ha minado
nosotros minamos - nosotros minaremos
ellos minan - ellos minarán
Busqué por la web sin saber bien que palabra encajaba y la mayoría de las busquedas hablan de la minería como un proceso. Me imágino que por un proceso ejecutado por un objeto y no una persona, por eso la utilizacion de la palabra minería en vez de minado.

lh1008 commented on translations/monero-core_es.ts:
```diff
@@ -610,27 +610,27 @@
         <location filename="../pages/Mining.qml" line="161"/>
         <source>Mining is only available on local daemons. Run a local daemon to be able to mine.&lt;br&gt;</source>
-        <translation>El minado sólo está disponible en nodos locales. Arranque un daemon local para poder minar.&lt;br&gt;</translation>
+        <translation>El minado solo está disponible en nodos locales. Arranque un daemon local para poder empezar.&lt;br&gt;</translation>
```
La minería.... .. para poder minar.

Mira lo que encontré en la terminología windows:
modelo de minería de datos
An object that contains the definition of a data mining process and the results of the training activity. For example, a data mining model may specify the input, output, algorithm, and other properties of the process and hold the information gathered during the training activity, such as a decision tree.
*Si miras lo que hace el deamon, es una ejecución de actividades que se desprenden de ramas de un arbol (mapa conceptual)

lh1008 commented on translations/monero-core_es.ts:
```diff
@@ -610,27 +610,27 @@
         <location filename="../pages/Mining.qml" line="173"/>
         <source>Stop mining</source>
-        <translation>Parar minado</translation>
+        <translation>Parar proceso de minado</translation>
```
Detener minería?

i3visio replied:
Más fácil. A veces uno se complica innecesariamente. :dancer:

lh1008 replied:
:+1: :)

lh1008 commented on translations/monero-core_es.ts:
```diff
@@ -704,35 +704,35 @@
         <location filename="../components/NewPasswordDialog.qml" line="100"/>
         <source>Please enter new password</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Introduzca nueva contraseña</translation>
```
Terminología windows...

- Escriba nueva contraseña
- Escribir nueva contraseña
- Escribir una nueva contraseña

lh1008 commented on translations/monero-core_es.ts:
```diff
@@ -704,35 +704,35 @@
         <location filename="../components/NewPasswordDialog.qml" line="157"/>
         <source>Please confirm new password</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Confirme la nueva contraseña</translation>
```
Terminología windows...

- Confirmar nueva contraseña
- Confirmar la nueva contraseña

erciccione commented:
hey @i3visio any update? can you implement @lh1008 's reviews?

lh1008 commented on translations/monero-core_es.ts:
```diff
@@ -2234,7 +2234,7 @@ For the case with Spend Proof, you don&apos;t need to specify the recipient addr
         <location filename="../main.qml" line="435"/>
         <source>Remaining blocks (local node):</source>
-        <translation type="unfinished">Nodos restantes (nodo local):</translation>
+        <translation>Nodos restantes (nodo local):</translation>
```
**Bloques** restantes (nodo local)

lh1008 commented on translations/monero-core_es.ts:
```diff
@@ -2444,47 +2444,47 @@ Descripción:</translation>
         <location filename="../main.qml" line="1108"/>
         <source>Warning</source>
-        <translation type="unfinished">Aviso</translation>
+        <translation>Aviso</translation>
```
Advertencia

lh1008 commented on translations/monero-core_es.ts:
```diff
@@ -1172,38 +1172,38 @@ La caché del monedero antiguo será renombrada para poder ser restaurada con po
         <location filename="../pages/Settings.qml" line="630"/>
         <source>Error: Filesystem is read only</source>
-        <translation type="unfinished">Error: el sistema de ficheros es de solo lectura</translation>
+        <translation>Error: el sistema de ficheros es de solo lectura</translation>
```
En la linea 2457 pusiste el siguiente:

"Error: el sistema de archivos es de solo lectura"...

Buscando en la terminologia windows ellos utilizan para filesystem = sistema de archivos

lh1008 commented on translations/monero-core_es.ts:
```diff
@@ -2444,47 +2444,47 @@ Descripción:</translation>
         <location filename="../main.qml" line="1113"/>
         <source>Warning: There&apos;s only %1 GB available on the device. Blockchain requires ~%2 GB of data.</source>
-        <translation type="unfinished"></translation>
+        <translation>Aviso: Solamente hay %1 GB disponibles en el dispositivo. La cadena de bloques requiere aproximadamente ~%2 GB de datos.</translation>
```
..."La cadena de bloques requiere ~%2 GB de espacio."

lh1008 commented on translations/monero-core_es.ts:
```diff
@@ -2444,47 +2444,47 @@ Descripción:</translation>
         <location filename="../main.qml" line="1115"/>
         <source>Note: There&apos;s %1 GB available on the device. Blockchain requires ~%2 GB of data.</source>
-        <translation type="unfinished"></translation>
+        <translation>Nota: Hay %1 GB disponibles en el dispositivo. La cadena de bloques requiere aproximadamente ~%2 GB de datos.</translation>
```
..."La cadena de bloques requiere ~%2 GB de espacio."

Este simbolo "~" ya dice que es un aproximado...

lh1008 commented on translations/monero-core_es.ts:
```diff
@@ -2444,47 +2444,47 @@ Descripción:</translation>
         <location filename="../main.qml" line="1117"/>
         <source>Note: lmdb folder not found. A new folder will be created.</source>
-        <translation type="unfinished">Nota: no se ha encontrado la carpeta lmdb. Se va a crear una nueva carpeta.</translation>
+        <translation>Nota: no se ha encontrado la carpeta lmdb. Se va a crear una nueva carpeta.</translation>
```
"Nota: carpeta lmdb no encontrada. Se va a crear una nueva carpeta."

erciccione commented:
Ping @i3visio

ordtrogen commented on translations/monero-core_es.ts:
```diff
@@ -1044,62 +1044,62 @@
         <location filename="../pages/Settings.qml" line="239"/>
         <source>Show advanced</source>
-        <translation type="unfinished">Mostrar avanzadas</translation>
+        <translation>Mostrar avanzadas</translation>
```
Creo q se hace referencia a los "settings". Es decir "Mostrar [configuraciones] avanzadas"

lh1008 replied:
Si puedes ser....ahora me pregunto lo mismo. Se pueden agregar palabras? 
Cuando lo lei...no entendi a que se referia Mostrar avanzadas. Le cambie a -avance- porque pense que se referia al log. Si son los "settings", Mostrar avanzadas sirve...lo que no podría saber es si otros al leer "Mostrar avanzadas" como lo interpreten. Tu que dices? "Mostrar configuraciones avanzadas"? Se puede o seria cambiar la definición?

i3visio replied:
Igual mejor lo dejamos como `Mostrar configuraciones avanzadas`.

ordtrogen commented on translations/monero-core_es.ts:
```diff
@@ -1044,62 +1044,62 @@
         <location filename="../pages/Settings.qml" line="351"/>
         <source>Node login (optional)</source>
-        <translation type="unfinished">Login del nodo (opcional)</translation>
+        <translation>Login del nodo (opcional)</translation>
```
Será más q sólo su inicio. "Login" sugiere proveer contraseña p ej, no?

lh1008 replied:
Si claro...login suena mejor. Le sugeri "Inicio" como buscando traducirlo tal cual y con el soporte de terminologia windows ...pero suena mejor "Login". En algunas traducciones le pregunte lo mismo a ErCiccione, con respecto a Stealth Address, "Direccion furtiva o sigilosa" suena muy extraño...Me dijo que era mejor dejarle stealth address mientras buscabamos un consenso en palabras técnicas...

i3visio commented on translations/monero-core_es.ts:
```diff
@@ -242,12 +242,12 @@
         <location filename="../components/HistoryTable.qml" line="50"/>
         <source>Tx key:</source>
-        <translation>Llave Tx:</translation>
+        <translation>Llave de Tx:</translation>
```
Sí, nos parece razonable. 

Con este tipo de cuestiones, uno se da cuenta de que podríamos avanzar hacia una guía de estilo propia del proyecto en la que hacer referencia a los conceptos que son intrínsecos de Monero. Aunque tomáramos como referencia las guías de Ubuntu por ejemplo (que están bastante completas por cierto, https://wiki.ubuntu.com/UbuntuSpanishTranslators) hay muchas cuestiones y conceptos que son propios del software y que no tienen estandarización y se podría crear una lista de traducciones conflicativas al estilo de las de Ubuntu (e. g., aquí: https://wiki.ubuntu.com/UbuntuSpanishTranslators/TraduccionesConflictivas) que fueran las recomendadas incluso para la documentación o artículos en español.

i3visio commented on translations/monero-core_es.ts:
```diff
@@ -310,123 +310,123 @@
         <location filename="../components/HistoryTableMobile.qml" line="152"/>
         <source>UNCONFIRMED</source>
-        <translation type="unfinished">NO CONFIRMADO</translation>
+        <translation>NO CONFIRMADO</translation>
```
Es más un pendiente de confirmar. Compro el «sin confirmar».

i3visio commented on translations/monero-core_es.ts:
```diff
@@ -947,12 +947,12 @@
         <location filename="../components/RemoteNodeEdit.qml" line="49"/>
         <source>Remote Node Hostname / IP</source>
-        <translation type="unfinished">Dirección IP o hostname del nodo remoto</translation>
+        <translation>Dirección IP o hostname del nodo remoto</translation>
```
Lo que no tenemos claro en este caso es el espaciado anterior y posterior. Por ejemplo, en los casos de `Firmar/verificar` está sin espaciado y en este, aunque también es un OR lógico, en el texto original no sigue el ejemplo. La verdad es que era una pequeña licencia poner el «o», pero es verdad que en cualquier caso hay que revertir el orden. Cambiando.

i3visio commented on translations/monero-core_es.ts:
```diff
@@ -989,22 +989,22 @@
         <location filename="../pages/Settings.qml" line="303"/>
         <source>Blockchain location</source>
-        <translation type="unfinished">Localización de la cadena de bloques</translation>
+        <translation>Localización de la cadena de bloques</translation>
```
Ubicación +1. :D

i3visio commented:
Sorry for the delay guys. Just updating the comments. Great job from our colleagues.

i3visio commented on translations/monero-core_es.ts:
```diff
@@ -1152,17 +1152,17 @@ La caché del monedero antiguo será renombrada para poder ser restaurada con po
         <location filename="../pages/Settings.qml" line="592"/>
         <source>Wallet log path: </source>
-        <translation type="unfinished">Ruta del log del monedero: </translation>
+        <translation>Ruta del log del monedero: </translation>
```
En este caso hace referencia al archivo de log del monedero (por ejemplo, `Wallet log path: /opt/monero-gui-v0.11.1.0/monero-wallet-gui.log`) y que es donde se registra cuándo se cargan los monederos, las claves, etc. Cambio `log`, por `fichero de registro`.

i3visio commented on translations/monero-core_es.ts:
```diff
@@ -704,35 +704,35 @@
         <location filename="../components/PasswordDialog.qml" line="88"/>
         <source>Please enter wallet password</source>
-        <translation>Por favor, introduzca la contraseña del monedero</translation>
+        <translation>Introduzca la contraseña del monedero</translation>
```
En este caso, hemos tomado la referencia de la guía de estilo de traductores de Ubuntu (https://wiki.ubuntu.com/UbuntuSpanishTranslators/Estilo), que recomienda lo siguiente sobre las órdenes:

```
Hay que tener especial cautela al traducir los infinitivos e imperativos. En inglés ambas formas se expresan igual, lo cual puede llevar a confusión. En español, las órdenes siempre se expresan en imperativo, nunca en infinitivo.

        Go to the home folder [en la descripción de un botón]
        Ir a la carpeta personal
                ⊗ Vaya a la carpeta personal
        
        Go to your home folder [en una lista con pasos para realizar algo]
        Vaya a su carpeta personal
                ⊗ Ir a su carpeta personal
```

Es una referencia, vaya, pero como va acompañada de fórmula de cortesía (ese `Please`), pues parece indicado ponerlo en imperativo:

```
En inglés, más que español, es usual encontrar estructuras de cortesía. En general, éstas no deben traducirse pues no aportan nada nuevo ni esencial y su carga de formalidad se expresa en español por otros medios (como un registro general más elevado y el uso de usted).

        Please, insert a blank disc.
        Introduzca un disco en blanco.
                ⊗ Por favor, introduzca un disco en blanco.
```

lh1008 replied:
Si en este caso tienes razon, la palabra "introduzca" sin el acompañamiento del "por favor". No suena bien decir "Por favor, introduzca...". Si se puede dejar "Introduzca" como traducción. Sugeri "Escribir"; "Introduzca" también sirve. Ese si lo tome de la terminologia windows. Soy muy nuevo en ubuntu y se que los términos son diferentes, a lo mejor tengo aún influencia de windows. Como dije mas arriba, esto no es windows, aunque nos podría servir el uso de la terminología, en ciertos casos, puede que este no lo sea. En este caso se los dejo a discreción de ustedes, estoy aprendiendo :).

erciccione commented:
thanks @i3visio can you squash those commits in just one?

erciccione commented:
This still need to be squashed, use `git rebase -i HEAD~3` (contact me if you need help)

i3visio commented:
We're not sure if we are doing it properly and just to avoid messing everything up, we prefer to ask.
Cloned from our own repo:
```
git clone https://github.com/i3visio/monero-gui
```
Rebased and squashesd into one single commit by doing:
```
$ git rebase -i HEAD~3
[detached HEAD 3ad5c9a] Update Spanish translations
 Date: Sat Feb 3 21:57:27 2018 +0100
 1 file changed, 216 insertions(+), 207 deletions(-)
Successfully rebased and updated refs/heads/master.
```
During this process, the log messages have been appropiately modified:
```
$ git log
commit 3ad5c9acf1b1562b5fb3a859a89b176fff3d7ad3
Author: i3visio <contacto@i3visio.com>
Date:   Sat Feb 3 21:57:27 2018 +0100

    Update Spanish translations
    
    Update non translated strings and remove commonly unneeded expressions in Spanish
    translations such as `Por favor,` and similar ones. `type="unfinished"` tags from
    already translated resources have also been removed.

commit 50fb2ddf699b6fdde298472aa268995ab2a68890
Merge: 5c66468 2061bc5
Author: luigi1111 <luigi1111w@gmail.com>
Date:   Tue Jan 30 00:28:19 2018 -0500

    Merge pull request #1101
    
    2061bc5 UnsignedTransaction: Removing the Priority enum accordingly to libwalletapi update
…
```
And then, we try to push this to our repo, we got it rejected:
```
$ git push origin master
[…] Authentication […]
To https://github.com/i3visio/monero-gui/
 ! [rejected]        master -> master (non-fast-forward)
error: failed to push some refs to 'https://github.com/i3visio/monero-gui/'
consejo: Updates were rejected because the tip of your current branch is behind
consejo: its remote counterpart. Integrate the remote changes (e.g.
consejo: 'git pull ...') before pushing again.
consejo: See the 'Note about fast-forwards' in 'git push --help' for details.
```
As we think that `git push origin master -f` is not desired, we try to do `git pull` to merge the changes which brings back the old commits which is what we just wanted to avoid:
```
$ git log
commit 62a719e664d706cd0784e9c2fc545e12cf004ad4
Merge: 3ad5c9a b4bc545
Author: i3visio <contacto@i3visio.com>
Date:   Wed Mar 14 10:22:42 2018 +0100

    Merge branch 'master' of https://github.com/i3visio/monero-gui

commit 3ad5c9acf1b1562b5fb3a859a89b176fff3d7ad3
Author: i3visio <contacto@i3visio.com>
Date:   Sat Feb 3 21:57:27 2018 +0100

    Update Spanish translations
    
    Update non translated strings and remove commonly unneeded expressions in Spanish
    translations such as `Por favor,` and similar ones. `type="unfinished"` tags from
    already translated resources have also been removed.

commit b4bc545ff43f8c000d1f58d1d597589b99ee5dde
Author: i3visio <contacto@i3visio.com>
Date:   Sun Mar 4 00:17:49 2018 +0100

    Update monero-core_es.ts

commit 776e8ee6fbc87200cc6529408963e4612e2334ed
Author: i3visio <contacto@i3visio.com>
Date:   Mon Feb 5 15:58:27 2018 +0100

    Update translations
    
    Remove `type="unfinished"` from translated resources.

commit 75bc37e192544ebcb01f20cdd838c29c71a91bf3
Author: i3visio <contacto@i3visio.com>
Date:   Sat Feb 3 21:57:27 2018 +0100

    Update Spanish translations
    
    Update non translated strings and remove commonly unneeded expressions in Spanish translations such as `Por favor,` and similar ones.

commit 50fb2ddf699b6fdde298472aa268995ab2a68890
Merge: 5c66468 2061bc5
Author: luigi1111 <luigi1111w@gmail.com>
Date:   Tue Jan 30 00:28:19 2018 -0500

    Merge pull request #1101
    
    2061bc5 UnsignedTransaction: Removing the Priority enum accordingly to libwalletapi update
…
```
Should we do `git pull` first and then squash including the merged commit? We're really sorry for the inconvenience!

erciccione commented:
@i3visio It was all good until the push. You need to use git push -f . Since the base is different after the rebase, you need to force push, no need to pull again.
So:

1. `git rebase -i HEAD~3`
2. change last two commits to `squash`
3. `git push -f origin master`
4. done

i3visio commented:
@erciccione, done! We were not sure if -f could crash the pull request. Thanks for the help!
